app/views/users/index.html.erb

```diff
@@ -37,10 +37,12 @@
               <span class="govuk-hint email-wrapper"><%= @user.email == member.email ? t('users.you') : member.email %></span>
             </h3>
             <% unless member.gds? %>
-              <span class="tick-cross-<%= member.cert_manager? ? 'tick' : 'cross' %>">
```

Comment:
This is repeats below maybe we can extract this into a partial and pass in what varies as parameters